.github/extract-runtime-versions.mjs

```diff
@@ -45,13 +48,14 @@ const miniflareVersion = miniflarePackage.version;
 const workerdVersionConstraint = miniflarePackage.dependencies.workerd;
 
 // 3. Load `workerd` `package.json`, getting `workerd` version
-const miniflareRequire = module.createRequire(miniflarePackagePath);
-const workerdMainPath = miniflareRequire.resolve("workerd");
-const workerdPackageJsonPath = findClosestPackageJson(workerdMainPath);
-assert(workerdPackageJsonPath !== undefined);
-const workerdPackageJson = await fs.readFile(workerdPackageJsonPath, "utf8");
-const workerdPackage = JSON.parse(workerdPackageJson);
-const workerdVersion = workerdPackage.version;
+const workerdMainPath = path.resolve(
+	__dirname,
+	"../packages/wrangler/node_modules/miniflare/node_modules/.bin/workerd"
+);
+
+const workerdVersion = execSync(workerdMainPath + " --version")
+	.toString()
+	.split(" ")[1];
```

Comment:
I'm definitely open to changing this, but my thinking was that it would catch instances where the binary is published with the wrong version number, which is sometimes more useful in the `workerd` case